controllers/feeds/getFeedsV2.js

```diff
@@ -39,8 +39,6 @@ const feedSwitch = async (feed) => {
     case 'main_feed_following':
       return 'main_feed_f2';
     case 'main_feed_f2':
-      return 'main_feed_broad';
-    case 'main_feed_broad':
       return 'main_feed';
     default:
       return 'main_feed_following';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change seems to be removing the 'main_feed_broad' case and redirecting 'main_feed_f2' directly to 'main_feed'. This could potentially alter the behavior of the code if there are any differences in how 'main_feed' and 'main_feed_broad' are processed. Please ensure that 'main_feed' can handle all cases that 'main_feed_broad' was intended for.
```diff
-    case 'main_feed_f2':
-      return 'main_feed_broad';
-    case 'main_feed_broad':
-       return 'main_feed';
+    case 'main_feed_f2':
+      return 'main_feed';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->